### PR TITLE
fix linking in pxz

### PR DESCRIPTION
**Pull request by bukzor**
See also:  https://github.com/Homebrew/linuxbrew/issues/293

### CODE REVIEW

**Comment by mistydemeo** [comment]
Has this been reported to the author of the software?

File: Library/Formula/pxz.rb [hunk 1]
```diff
@@ -34,3 +34,16 @@ def install
  		if (m == MAP_FAILED) {
  			perror("mmap failed");
  			exit(EXIT_FAILURE);
+diff --git a/Makefile.orig b/Makefile
```
Comment (mikemcquaid) [inline Library/Formula/pxz.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

**Comment by xu-cheng** [comment]
Is this problem related to Linux only? See https://github.com/Homebrew/linuxbrew/issues/293#issuecomment-75915521

File: Library/Formula/pxz.rb (hunk 1 above)
Comment (bukzor) [inline Library/Formula/pxz.rb]: This is already fixed in upstream HEAD: https://github.com/jnovy/pxz/blob/master/Makefile#L15

The maintainer is dragging his feet making a fresh release though: https://github.com/jnovy/pxz/issues/21

**Comment by jacknagel** [comment]
Passing on this patch since the issue doesn't seem to affect OS X.

**Pull request closed without merging**